### PR TITLE
support HMAC-SM3

### PR DESCRIPTION
**Pull request by Gsealy**
support for HMAC-SM3

### CODE REVIEW

**Comment by dghgit** [comment]
Thanks for the patch, now merged with minor changes. See https://www.bouncycastle.org/betas 166b13 or later.

**Comment by Gsealy** [comment]
Thanks

File: prov/src/main/java/org/bouncycastle/jcajce/provider/digest/SM3.java [hunk 1]
```diff
@@ -29,6 +33,27 @@ public Object clone()
         }
     }
 
+    /**
+     * SHA1 HMac
```
Comment (Gsealy) [inline prov/src/main/java/org/bouncycastle/jcajce/provider/digest/SM3.java:37]: @dghgit sorry.. I was forget change javadoc, plz help me modify this, tks! 😃

Comment (dghgit) [reply prov/src/main/java/org/bouncycastle/jcajce/provider/digest/SM3.java:37]: I guess I forgot to read it! Should be updated soon.

**Pull request closed without merging**